apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/MySqlStressTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#if NET45 || NET451 || NET452
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Google.Cloud.Spanner.V1.Logging;
+using MySql.Data.MySqlClient;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.IntegrationTests
+{
+    [Collection("Spanner Integration Tests")]
```

Comment:
Yes, they all need to be in the same collection to avoid having other tests run concurrently and possibly cause side effects.
Tests within the same collection are run sequentially.